docker-compose.yml

```diff
@@ -161,7 +161,7 @@ services:
       - CLAMAV_PORT=3310
       - CLAMAV_HOST=clamav
       - CLAMAV_DEBUG_MODE_ENABLED=true
-      - CLAMAV_SCANNING_ENABLED=true
+      - CLAMAV_SCANNING_ENABLED
```

Comment:
definitely a good change!

Reply:
I've updated the other docker-compose files too